### ISSUE TITLE
Command "normalize" is not defined

### ISSUE THREAD

**Issue by localheinz** [localheinz/composer-normalize-action#11]
#### Steps required to reproduce the problem

1. Install action via https://github.com/marketplace/actions/localheinz-composer-normalize-action
2. Trigger action by pushing a commit into a repository that contains `composer.json`

#### Expected Result

* Build fails or passes, but with output from running `composer normalize`

#### Actual Result

* Build fails with

```
  [Symfony\Component\Console\Exception\CommandNotFoundException]  
  Command "normalize" is not defined.
```

It appears that somehow `localheinz/composer-normalize` doesn't end up in the image.

#### Examples

* https://github.com/localheinz/composer-normalize-action-example/pull/3/checks?check_run_id=135292799
* https://github.com/localheinz/composer-normalize-action-example/pull/2/checks?check_run_id=135292722
* https://github.com/localheinz/composer-normalize-action-example/pull/1/checks?check_run_id=135292710

**Comment by localheinz** [comment]
Fixed by #18.

**Comment by derrabus** [comment]
I ran into the same problem when I tried to upgrade the action to 0.7.0. Version 0.5.2 works fine.

<img width="570" alt="Bildschirmfoto 2020-04-11 um 16 06 24" src="https://user-images.githubusercontent.com/1506493/79047056-ae14a480-7c14-11ea-9376-8d8b25e6b435.png">
